templates/reviews_body.html

```diff
@@ -1,13 +1,62 @@
 {% if not reviews %}
-    <p>There are currently no reviews for this room</p>
-
+<p>There are currently no reviews for this room</p>
+<hr>
 {% else %}
-    {% for review in reviews %}
-        <h3> <strong>Review #{{ loop.index }}</strong></h3>
-        <h5>Posted on {{ get_time_string_from_utc_dt(review.date) }} </h5>
-        <h6>Rating: {{ review.rating }} out of 5</h6>
-        <p> {{ review.content }} </p>
-        <hr>
-    {% endfor %}
-
-{% endif %}
\ No newline at end of file
+{% for review in reviews %}
+<!--<h3><strong>Review #{{ loop.index }}</strong></h3>-->
+<h5>Posted on {{ get_time_string_from_utc_dt(review.date) }}
+    {% if review.net_id %}
+    by <em>{{ review.net_id }}</em>
+    {% endif %}
+</h5>
+<h6>Rating: {{ review.rating }} out of 5</h6>
+<p> {{ review.content }} </p>
+
+<hr>
+{% endfor %}
+
+{% endif %}
+
+
+<form action="{{ url_for('submit_review') }}"
+      method="post"
+      enctype="multipart/form-data"
+      id="submit-review-form">
+    <div style="font-size: 18px; margin-bottom: 0.5rem">If you were in
+        <strong>{{ building_name }}, Room {{ room_number }}</strong>, please submit your review below 😊
+    </div>
+    <input type="text" name="building-name" value="{{ building_name }}" hidden>
+    <input type="text" name="room-number" value="{{ room_number }}" hidden>
+    <div class="form-group">
+        <label for="overall-rating">Overall rating</label>
+        <select class="form-control" id="overall-rating" name="overall-rating" required>
+            <option value="None" selected="selected"></option>
+            <option>0</option>
+            <option>1</option>
+            <option>2</option>
+            <option>3</option>
+            <option>4</option>
+            <option>5</option>
+        </select>
+    </div>
+    <div class="form-group">
+        <label for="written-review">Add your written review</label>
+        <textarea class="form-control" id="written-review" rows="3" name="written-review" required></textarea>
```

Comment:
confirming that the review can be any length? is the DB able to store really long reviews?

Reply:
max length of 5000 sounds good. will the max length be displayed on the front end?

Reply:
> max length of 5000 sounds good. will the max length be displayed on the front end?

yes, currently does so

Reply:
done